src/Umbraco.Infrastructure/HostedServices/RecurringHostedServiceBase.cs

```diff
@@ -29,18 +31,24 @@ public abstract class RecurringHostedServiceBase : IHostedService, IDisposable
         /// <summary>
         /// Initializes a new instance of the <see cref="RecurringHostedServiceBase"/> class.
         /// </summary>
-        /// <param name="period">Timepsan representing how often the task should recur.</param>
-        /// <param name="delay">Timespan represeting the initial delay after application start-up before the first run of the task occurs.</param>
-        protected RecurringHostedServiceBase(TimeSpan period, TimeSpan delay)
+        /// <param name="logger">Logger.</param>
+        /// <param name="period">Timespan representing how often the task should recur.</param>
+        /// <param name="delay">Timespan representing the initial delay after application start-up before the first run of the task occurs.</param>
+        protected RecurringHostedServiceBase(ILogger logger, TimeSpan period, TimeSpan delay)
```

Comment:
This is a breaking change, we need an alernate ctor without a logger.

Reply:
I have amended this in my PR: https://github.com/umbraco/Umbraco-CMS/pull/12161